spec/support/bad_replicants.rb

```diff
@@ -2,5 +2,6 @@
   'Googlebot' => 'Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)',
   'Bingbot' => 'Mozilla/5.0 (compatible; bingbot/2.0; +http://www.bing.com/bingbot.htm)',
   'Yahoo! Slurp' => 'Mozilla/4.0 (compatible; Yahoo! Slurp; http://help.yahoo.com/help/us/ysearch/slurp)',
-  'Blank String' => '' # for the moment we're treating a blank user agent string as a bot
+  'Blank String' => '', # for the moment we're treating a blank user agent string as a bot
+  'Blank String with more spaces' => '    '
```

Comment:
These are tested in `voight_kampff_spec.rb`

Good bots are tested with the strings in `humans.rb`. The strings in that file already contain spaces/whitespace

Reply:
:+1: